### PR TITLE
<MessageBoxMarketerialLayout> - add prop to disable primary button

### PR DESCRIPTION
**Pull request by liorur**
### What changed
add prop to disable primary button in MessageBoxMarketerialLayout

- [v ] Change is tested
- [v ] Change is documented
- [v ] Build is green
- [ ] Change of UX is approved by [wuwa](https://github.com/wuwa) or [milkyfruit](https://github.com/milkyfruit)

### CODE REVIEW

**Comment by wuwa** [comment]
yes, although would prefer to see use case.

**Comment by liorur** [comment]
@wuwa  - the use case is not to allow continuing before accepting terms:
screen shot 2018-11-21 at 20 43 25

**Comment by wuwa** [comment]
cool, approved

File: stories/MessageBox/AnnouncementExamples/DisabledAction.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {MessageBoxMarketerialLayout} from 'wix-style-react/MessageBox';
+
+export default () => (
```
Comment (yardnsm) [inline stories/MessageBox/AnnouncementExamples/DisabledAction.js]: Please add an eyes test for this example.

https://github.com/wix/wix-style-react/blob/master/src/MessageBox/MessageBox.e2e.js#L80

**Pull request merged**